packages/data-format/index.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable no-unused-vars */
+import Big from 'big.js'
+Big.RM = 0
+Big.NE = -19
+
+export const fromDripToCfx = number => {
```

Comment:
Better name this `str` or `strOrNum`, since we use big.js to deal with big int.

The naming `number` may mislead consumers to convert the input to number before passing to this function.

Reply:
done